### ISSUE TITLE
Exception after restarting the server while client is connected

### ISSUE THREAD

**Issue by ralscha** [cettia/cettia-java-server#31]
This behaviour can be reproduces with any of my example applications (https://github.com/ralscha/cettia-demo).

1. Start server
2. Open browser (http://localhost:8080/)
3. Restart server (while browser is still open)

The server throws this exception. 
```
Caused by: java.lang.NullPointerException: null
	at java.util.concurrent.ConcurrentHashMap.get(ConcurrentHashMap.java:936) ~[na:1.8.0_181]
	at io.cettia.transport.http.HttpTransportServer.on(HttpTransportServer.java:171) ~[cettia-server-1.1.0-Beta1.jar:na]
	at io.cettia.transport.http.HttpTransportServer.on(HttpTransportServer.java:61) ~[cettia-server-1.1.0-Beta1.jar:na]
	at io.cettia.asity.action.AbstractActions.fireOne(AbstractActions.java:113) ~[asity-action-2.0.0.jar:na]
	at io.cettia.asity.action.AbstractActions.fire(AbstractActions.java:86) ~[asity-action-2.0.0.jar:na]
	at io.cettia.asity.bridge.spring.webflux5.AsityHandlerFunction.lambda$handle$0(AsityHandlerFunction.java:55) ~[asity-bridge-spring-webflux5-2.0.0.jar:na]
	at reactor.core.publisher.FluxDoFinally$DoFinallySubscriber.runFinally(FluxDoFinally.java:156) ~[reactor-core-3.1.8.RELEASE.jar:3.1.8.RELEASE]
	... 50 common frames omitted
```

It's not a big deal, because the client is able to reconnect to the server.

**Comment by flowersinthesand** [comment]
Yes. It's not a big deal since `abort` request handling at HttpTransportServer.java:171 is just to help detect disconnection on the server-side.

I'll fix it in 1.1.0. Thanks for the report.

**Comment by flowersinthesand** [comment]
I fixed and deployed a 1.1.0-SNAPSHOT. Could you test the fix when you get a chance just to be sure?

**Comment by ralscha** [comment]
Looks good. Exception no longer appears.
